.github/workflows/test-suite.yml

```diff
@@ -0,0 +1,49 @@
+name: Test Suite
+
+on:
+  pull_request:
+    branches: [develop, trunk]
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: PHP ${{ matrix.php }} WP ${{ matrix.wp }}
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    env:
+      WP_TESTS_DIR: /tmp/wordpress-tests-lib
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.2', '7.4', '8.0']
```

Comment:
It doesn't have to delay this (in case it turns out there's a problem), but I'd like to look at adding 8.1 and 8.2 as a followup.